src/main/scala/scalaz/parsers/tc/Alternative.scala

```diff
@@ -12,12 +15,21 @@ object Alternative {
         f1.orElse(f2)
     }
 
-  implicit def eitherAlternative[R]: Alternative[Either[R, ?]] =
-    new Alternative[Either[R, ?]] {
-      override def or[A](f1: Either[R, A], f2: => Either[R, A]): Either[R, A] =
+  implicit def eitherAlternative[L]: Alternative[Either[L, ?]] =
+    new Alternative[Either[L, ?]] {
+      override def or[A](f1: Either[L, A], f2: => Either[L, A]): Either[L, A] =
```

Comment:
Why were the `L`s changed to `R`s ?

Reply:
It's vice versa. I've incorrectly put `R` on the left side. Now it's `L` on the left side.